### DISCUSSION TITLE
Interface Settings

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17163] [Feature Requests]
It would be nice to have “Interface Settings” similar to “Project Settings”, where we can specify default values for each interface type.

Example: For every text input I am always enabling `trim the start and the end` over and over again. Would be nice to set it only once.

Alternatively we could do it via environmental values.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
